docs/ad/diagnostics/charged_particle_radiography/synthetic_radiography.rst

```diff
@@ -1,12 +1,12 @@
-.. _charged_particle_radiography:
+.. _synthetic_radiography:
 
 *************************************
```

Comment:
```suggestion
**************************************
```

Reply:
This right here is what I absolutely hate about restructuredtext.

Reply:
I'm just going to add myst-parser to our doc dependencies so that we may also write these in Markdown. Hold my green tea.